src/main/java/Game.java

```diff
@@ -2,16 +2,26 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 import java.util.HashMap;
+import java.util.logging.Logger;
 
 public class Game {
     private static BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
     private static HashMap<String, Move> moves = new HashMap<String, Move>();
+    public static Field field = new Field(10);
 
     static {
         moves.put("u", Move.UP);
         moves.put("d", Move.DOWN);
         moves.put("l", Move.LEFT);
         moves.put("r", Move.RIGHT);
+        field = null;
+        field = new Field(5);
+        field = null;
+        try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00108)